lua/weapons/pac_cw_extrema_ratio/cl_init.lua

```diff
@@ -0,0 +1,6 @@
+include( "shared.lua" )
+
+SWEP.PrintName            = "PAC_CW_EXTREMA_RATIO"
```

Comment:
either line them up, or put the `=` one space from the variable.

Reply:
I agree, just gonna be a pain to refactor the entire repository. I'll look into it for this file however since Phatso wants me to change it to not be Extrema. (We're gonna try for the CS:S Knife if possible)